### PR TITLE
PLAT-10 Navbar

### PR DESCRIPTION
**Pull request by gimenomarc**

### CODE REVIEW

File: src/Components/Btn.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
```
Comment (EsteveSegura) [inline src/Components/Btn.jsx:1]: This component should not exists, whats the purpose of wraping a component button inside another component button?

File: src/Styles/Btn.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.btn-container {
```
Comment (EsteveSegura) [inline src/Styles/Btn.css:1]: You should never apply a global css rule like that.

What would happen if you dont want the 30px?
Create more components?
start writing multiple rules?

**Comment by EsteveSegura** [comment]
The component folder should be 
--> Components
----- > NavBar
--------- > NavBar.jsx
--------- > NavBar.css

File: src/Components/Btn.jsx (hunk 1 above)
Comment (gimenomarc) [reply src/Components/Btn.jsx:1]: Btn component removed.

**Pull request merged**